pyscitt/pyscitt/cli/retrieve_signed_claims.py

```diff
@@ -52,11 +53,22 @@ def cli(fn):
         type=Path,
         help="Folder containing JSON parameter files of SCITT services to trust",
     )
+    parser.add_argument(
+        "--dont-embed-receipt",
+        action="store_true",
+        default=False,
```

Comment:
```suggestion
        default=True,
```

Reply:
It doesn't make sense to set the default value for this flag to `True`: how can users set it to `False` if they pass the flag? By default, it will be False and adding the CLI argument will set it to True.